proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -91,12 +97,15 @@ message ListUsersRequest {
   // The first page is returned if this is empty.
   string page_token = 2;
   bool reverse = 3;
+  // request with unspecified user state will return users with both user state
+  UserState user_state = 4;
 }
 
 message ListUsersResponse {
   repeated User users = 1;
   // There are no more pages if this is empty.
   string next_page_token = 2;
+  int32 total_size = 3;
```

Comment:
Unless we are adding `total_size` to every `List*` endpoint, this breaks our convention for `List*` endpoints. If we do want this data, it should likely be a separate RPC.

Also if we do expose this data somehow, it should be a `uint32`.

Reply:
The idea of adding `int32 total_size` in the response of Listing endpoints was came from this [Google Cloud APIs suggested Common design patterns](https://cloud.google.com/apis/design/design_patterns#list_pagination), and I agree that `uint32` is actually a better data type in this case.

For the concern of the convention, I agree that we should keep the convention for the Listing endpoints. In the beginning I was thinking about adding some new endpoints for the information needed, but because of the concern of API bloat, I think adding the `total_size` to all of the Listing endpoints might be a better approach, and I have created a separated PR for it (#734), and this PR will only for the `ListUsers` with user state filter.

cc @robbertvanginkel @mfridman as they also have the concern of API bloat